src/backend/expungeservice/test/test_expunger.py

```diff
@@ -20,16 +20,51 @@ def test_statute():
 
 def get_disp():
     date = datetime.date(1996, 1, 1)
-    disp_type = DispositionType.NO_CONVICTION
+    disp_type = DispositionType.CONVICTED
     return Disposition(disp_type, date)
 
+def get_qualifying_charge():
+    # Fix me with correct charge
+    disp = get_disp()
+    statute = Statute(113, 45, 5, 'd')
+    level = CrimeLevel('Infraction', 'A')
+    return Charge('Qualifiying Charge', statute, level,
+                  datetime.date(1995, 1, 1), disp)
+
+def get_unqualifying_charge():
+    # Fix me with correct charge
+    disp = get_disp()
+    statute = Statute(113, 45, 5, 'd')
+    level = CrimeLevel('Felony', 'A')
+    return Charge('Unqualifying Charge', statute, level,
+                  datetime.date(1995, 1, 1), disp)
+
 def test_expunger_classes():
     disp = get_disp()
     statute = Statute(113, 45, 5, 'd')
-    level = Level('Felony', 'A')
+    level = CrimeLevel('Felony', 'A')
     charges = [
-        Charge('Charge Name', statute, level,
-               datetime.date(1995, 1, 1), disp)
+        get_qualifying_charge(),
+        get_unqualifying_charge(),
     ]
-    cases = [Case(charges, CaseState.OPEN), 100.50]
+    cases = [
+        Case(charges, CaseState.OPEN, 100.50),
+        Case(charges, CaseState.CLOSED, 100.50),
+    ]
+    client = Client('John Doe', datetime.date(1970, 1, 1), cases)
+    assert(client.num_charges() == 4)
+
+def test_expunger():
+    charges = [
+        get_qualifying_charge(),
+        get_unqualifying_charge(),
+    ]
+    cases = [Case(charges, CaseState.OPEN, 100.50)]
     client = Client('John Doe', datetime.date(1970, 1, 1), cases)
+    record_analyzer = RecordAnalyzer(client)
+    time_elig_result = record_analyzer.analyze()
+
+    # TODO add individual test cases that test different paths in the decision tree
+    assert(charges[0].type_elig_result.code == ResultCode.FURTHER_ANALYSIS)
+    assert(charges[1].type_elig_result.code == ResultCode.INELIGIBLE)
```

Comment:
This is testing the crime level and not the statutue? I thought it was the statute that is used to determine expungeability?

Reply:
I see you have it to check the level and then statutes... Does it only matter if the level is a felony class A? It seems statutes may have different levels, is that a correct assumption?

Reply:
> Does it only matter if the level is a felony class A? 

Yes in this context. The tests are just following the checks in the flowchart.

> It seems statutes may have different levels, is that a correct assumption?

I'm not sure about that. We'd have to check with Michael.

Reply:
@michaelatcommunitylaw Can there be different level's associated to a charge?

Reply:
Michael responded in slack and said that there are not different levels associated.